src/actions/sendContactForm.ts

```diff
@@ -0,0 +1,52 @@
+"use server";
+
+import nodemailer from "nodemailer";
+
+interface SendEmailProps {
+  name: string;
+  email: string;
+  subject: string;
+  message: string;
+  token: string;
+}
+
+export async function sendContactForm({ name, email, subject, message, token }: SendEmailProps) {
+  if (!token) {
+    return { success: false, message: "CAPTCHA verification failed" };
+  }
```

Comment:
I presume that `sendContactForm` is a sort of Back-End form and it should not care about some CAPTCHA token validation. From my point of view it should just accept incoming arguments and send an email. Let's put this token validation on client side, please.

Reply:
It makes sense. Updated